direct-query-core/src/main/java/org/opensearch/sql/directquery/rest/model/WriteDirectQueryResourcesRequest.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.sql.directquery.rest.model;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.Map;
+
+/*
+ * @opensearch.experimental
+ */
+@Data
+@NoArgsConstructor
+public class WriteDirectQueryResourcesRequest {
+  private String dataSource;
+  private DirectQueryResourceType resourceType;
```

Comment:
Is this class form earlier PRs? Is this going to be a list of all resource types from all datasource types?